samples/package.json

```diff
@@ -18,6 +18,7 @@
     ]
   },
   "scripts": {
+    "fix": "eslint --fix '*.js'",
```

Comment:
I'd rather us not add this, and just rely on `npm run fix` from the root of the project.